test/sentry/logger_handler_test.exs

```diff
@@ -154,7 +154,7 @@ defmodule Sentry.LoggerHandlerTest do
       assert %KeyError{} = event.original_exception
       assert [exception] = event.exception
       assert exception.type == "KeyError"
-      assert exception.value == "key :foo not found in: []"
+      assert exception.value == "key :foo not found in:\n\n    []"
```

Comment:
That's unrelated to the translation: https://github.com/elixir-lang/elixir/pull/14269